core/store/models/run_input.go

```diff
@@ -0,0 +1,68 @@
+package models
+
+import (
+	"fmt"
+
+	"github.com/tidwall/gjson"
+)
+
+// RunInput represents the input for performing a Task
+type RunInput struct {
+	jobRunID ID
+	data     JSON
+	status   RunStatus
+	err      error
+}
```

Comment:
When is 'err' set for the RunInput?  Its not clear from this file

Reply:
You're right! Removed.